### ISSUE TITLE
Sprinkled [TODO] tags everywhere.

### ISSUE THREAD

**Issue by opinionatedguide** [VegaDeftwing/opinionatedguide#3]
For anyone looking to contribute, weather for fun or for a Hacktoberfest T-Shirt (I'll only be approving reasonably sized pulls- no cheating) there's a few hundred [TODO] tags in the various markdown files, especially in all the engineering chapters. A lot of it is pretty easy-picking too.

... please don't just copy paste a Wikipedia article though.

**Comment by VegaDeftwing** [comment]
During the month of October I will **not** be accepting **minor** changes. Basically, you can't just fix a typo and open a pull. Normally this would be fine (albeit a bit weird?) but because I want to keep this repo available to all and have the commits count for Hacktoberfest I ask that you please submit a significant sized pull - like at least a paragraph, but that's still a bit shallow. This is especially true if you try to do multiple pulls. If it's because each one is justifiable in it's own right, sure, but like, don't do two one paragraph pulls. Basically, don't abuse the spirit of Hacktoberfest.

Thank you for contributing <3

**Comment by sesam** [comment]
I had the understanding that the Hacktoberfest's "scoring" just checks that PR:s aren't marked as spam. So, folks, don't make silly PRs m'kay :)

**Comment by cetteSara** [comment]
Hi ! I was wondering, the [TODO]s that are up for grabs are, for example, like this page: https://opinionatedguide.github.io/#/Design/d1-basics ?

I think I could be able to fill some infos in the Design section. Is there something in particular that you're looking for ?

**Comment by VegaDeftwing** [comment]
@cetteSara , yep, [TODO] tags on completely empty pages like that or any of the hundreds throughout the guides that are more specific, mostly just needing expanded on.

If you could flesh out the design section some that'd be awesome! I'm not the best with design or art myself, so I'm not 100% sure what even should go there, probably just things that are applicable to both traditional and digital art - concepts like contrast, color pallets, etc. so that the more specific things for traditional and digital art can be kept on their respective pages.

If you think that more pages need to be added or content in the design section needs reorganized you can also add new files, just update _sidebar.md
